package.json

```diff
@@ -19,7 +19,7 @@
   "version": "5.1.1",
   "engines": {
     "vscode": "^1.63.0",
-    "node": "^16.13.0"
+    "node": "^21.0.0"
```

Comment:
I'd rather not update this as it's not needed by the code or any of the packages.